src/Domain/RegularGivingNotifier.php

```diff
@@ -32,6 +32,7 @@ public function notifyNewMandateCreated(
                 'templateKey' => 'donor-mandate-confirmation',
                 'recipientEmailAddress' => $donorAccount->emailAddress->email,
                 'params' => [
+                    'donorName' => $donorAccount->donorName->fullName(),
```

Comment:
not related to other changes but necessary bug fix to be able to send this email.